<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QPushButton" name="botaoAcessar">
     <property name="text">
      <string>Acessar </string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="labelTempo">
     <property name="text">
      <string>Tempo de operação</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>ID do projeto</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="inputTempo"/>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QPushButton" name="botaoCancelar">
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="inputProjeto"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelUsuario">
     <property name="text">
      <string>Usuário</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QPushButton" name="botaoOk">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="inputUsuario"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="labelID">
     <property name="text">
      <string>ID da maquina</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="inputID"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
